docs/source/index.rst

```diff
@@ -0,0 +1,40 @@
+.. TracPro documentation master file, created by
+   sphinx-quickstart on Mon Nov  2 13:52:53 2015.
+
+Welcome to TracPro's documentation!
+===================================
+
+TracPro is a minimal, generic dashboard for polls in any `RapidPro <https://app.rapidpro.io/>`_ org, such as for a real-time monitoring service for education. TracPro is for simple dashboards that help to create information loops, not for advocacy or flashy visuals. TracPro is open source, anyone can build their own dashboard. Deployment requires advanced Linux systems administration experience, but once running it can be configured and used by anyone.
+
+TracPro was built for UNICEF by `Nyaruka <http://nyaruka.com/>`_ and `Caktus Group <https://www.caktusgroup.com/>`_.
+
+
+Developer Guides:
```

Comment:
On second thought, I think we'll need a third directory like `users/` for things like `technical-overview` and possibly `getting-started`.